### PR TITLE
*: Fix issue#959

### PR DESCRIPTION
**Pull request by zimulala**
fix issue #959

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

File: meta/autoid/autoid_test.go [hunk 1]
```diff
@@ -61,4 +61,27 @@ func (*testSuite) TestT(c *C) {
...
+	c.Assert(err, IsNil)
+	id, err = alloc.Alloc(1)
+	c.Assert(err, IsNil)
+	c.Assert(id, Equals, int64(11))
+	err = alloc.Rebase(1, int64(3010))
+	c.Assert(err, IsNil)
+	id, err = alloc.Alloc(1)
+	c.Assert(err, IsNil)
+	c.Assert(id, Equals, int64(3011))
+
```
Comment (zxylvlp) [inline meta/autoid/autoid_test.go]: empty line.

**Comment by zimulala** [comment]
PTAL @zxylvlp

**Comment by zxylvlp** [comment]
LGTM

**Pull request merged**